daemon/graphdriver/quota/projectquota.go

```diff
@@ -56,10 +58,15 @@ import (
 	"path/filepath"
 	"unsafe"
 
+	"errors"
+
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sys/unix"
 )
 
+// ErrQuotaNotSupported indicates if were found the FS didn't have projects quotas available
+var ErrQuotaNotSupported = errors.New("Filesystem does not support, or has not enabled quotas")
```

Comment:
errors should start with lower case letter and the comma seems out of place there

Reply:
I'd like to use this error in other packages, if you see the work in https://github.com/moby/moby/pull/35231, it uses this. I can export the error in another PR, but it's already being used here.

Reply:
Sorry, I meant the error string `filesystem does not support`